### DISCUSSION TITLE
How do I print double curly  braces

### DISCUSSION THREAD

**Discussion by hueyvle** [honkit/honkit#323] [Q&A] [answered]
hi guys, 
I have this line

{{library:alias}}

How do I get honkit to output exactly that?

Thanks

**Comment by azu** [comment, marked as answer]
```
{% raw %}
{{library:alias}}
{% endraw %}
```

You can use `raw` tag.
https://honkit.netlify.app/templating/#escaping

**Comment by hueyvle** [comment]
Solved my issue.  
Thank you so much!
